web-console/knikubevirt/NoStorageClassAvailable/NoStorageClassAvailable.md

```diff
@@ -0,0 +1,35 @@
+# Storage class isn’t defined
+
+https://github.com/openshift/openshift-origin-design/issues/249
+https://jira.coreos.com/browse/CNV-2842
```

Comment:
I'd pull these links out of here and add them to the description. (prob drop the jira one altogether)